meson.build

```diff
@@ -113,9 +118,9 @@ cdata.set('EXV_HAVE_XMP_TOOLKIT', expat_dep.found())
 cdata.set('EXV_HAVE_BROTLI', brotli_dep.found())
 cdata.set('EXV_HAVE_ICONV', iconv_dep.found())
 cdata.set('EXV_HAVE_LIBZ', zlib_dep.found())
-cdata.set('EXV_USE_CURL', curl_dep.found())
+cdata.set('EXV_ENABLE_WEBREADY', get_option('webready'))
+cdata.set('EXV_USE_CURL', get_option('webready') and curl_dep.found())
```

Comment:
get_option('webready') can be removed here.

curl_dep.found() always returns false with
curl_dep = dependency('', disabler: true, required: false)

Reply:
Good call, it doesn't make sense since the last change. Thanks